engine/gamesys/src/gamesys/components/comp_mesh.cpp

```diff
@@ -303,6 +305,59 @@ namespace dmGameSystem
         component->m_ReHash = 0;
     }
 
+    // Initial max/min values for mesh bounds
+    static const dmVMath::Point3 BOUNDS_MIN_EMPTY(1e10,1e10,1e10);
+    static const dmVMath::Point3 BOUNDS_MAX_EMPTY(-1e10,-1e10,-1e10);
+
+    static void UpdateMeshBounds(dmBuffer::HBuffer hbuffer, dmhash_t stream_name) {
+
+        // get stream and buffer properties
+        void* vertices_data;
+        uint32_t vertex_count;
+        uint32_t vertex_components;
+        uint32_t stride;
+
+        dmBuffer::Result r = dmBuffer::GetStream(hbuffer, stream_name, &vertices_data,  &vertex_count, &vertex_components, &stride);
+        if (r != dmBuffer::RESULT_OK) {
+            dmLogError("Error calculating mesh bounds for culling. Invalid buffer or stream: '%s'.", dmHashReverseSafe64(stream_name) );
+            return;
+        }
+
+        // set up initial values
+        dmVMath::Point3 boundsMin = BOUNDS_MIN_EMPTY;
```

Comment:
That's the best solution i could come down with considering all other requirements for this feature.

boundsMin/Max are initialized to a _specific_ far away value. In case of error, the bounds are left to their initial values.  Later, in RenderListFrustumCulling(), if the boundsMin/Max have not been updated entry's m_Visibility is not udated.